csharp/tools/tracing-config.lua

```diff
@@ -226,6 +226,11 @@ function RegisterExtractorPack(id)
             prepend = { '--compiler', '"${compiler}"' },
             order = ORDER_BEFORE
         }),
+        CreatePatternMatcher({ '^csc$' }, MatchCompilerName,
```

Comment:
Why not just include `'^csc$'` in the list above?

Reply:
Due to the quotation in the above.